ui/src/app/shared/components/graph/label.tsx

```diff
@@ -9,6 +9,30 @@ export const formatLabel = (label: string) => {
         return <tspan>{label}</tspan>;
     }
     if (label.length <= maxPerLine * 2) {
+        const split = label.split('-');
+        if (split.length >= 2) {
```

Comment:
maybe we should have a unit test for this logic now?

Reply:
otherwise - nice work

Reply:
Absolutely, I'll work on some unit tests

Reply:
Tests done